### PR TITLE
[Sprite] Beta2 Variant Patch icon chores

### PR DESCRIPTION
**Pull request by chaosgrimmon**
## What are the changes the user will see?
No user-side changes.

## Why am I making these changes?
#3470 did not update any icons in the `variant` folders. (They should appear after the folder for Gen `9`)
image

On top of various fixes necessitated by that batch of icons, _other_ icons also needed some updates.

Although the icons in these folders aren't user-facing, and do not have an immediate impact, collecting them all in one place is a crucial part of efficiently generating icon sheets.

## What are the changes from a developer perspective?
The images in these folders now match the art team's desired icon style goals more closely.

## How to test the changes?
No changes should be observed in Starter Select, or any other place where icons are used.

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
~~- [ ] Have I considered writing automated tests for the issue?~~
~~- [ ] If I have text, did I add make it translatable and added a key in the English language?~~
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
~~- [ ] Are the changes visual?~~
~~  - [ ] Have I provided screenshots/videos of the changes?~~

### CODE REVIEW

File: public/images/pokemon/icons/variant/3/256_2.png
Comment (pom-eranian) [inline public/images/pokemon/icons/variant/3/256_2.png:1]: 256_2  male is incorrect
image
https://discord.com/channels/1125469663833370665/1274883292897738774/1277269643534401608

File: public/images/pokemon/icons/variant/3/255_2.png
Comment (pom-eranian) [inline public/images/pokemon/icons/variant/3/255_2.png:1]: 255_2 male is incorrect
image
https://discord.com/channels/1125469663833370665/1274883292897738774/1277269643534401608

File: public/images/pokemon/icons/variant/4/424_2.png
Comment (pom-eranian) [inline public/images/pokemon/icons/variant/4/424_2.png:1]: This one is quite different from the sprite. Up to art's discretion to fix or not.
6e70ffed-d93b-48ae-83ae-94c28c512abf

File: public/images/pokemon/icons/variant/8/867_2.png
Comment (pom-eranian) [inline public/images/pokemon/icons/variant/8/867_2.png:1]: This one doesn't match.
0bc1bf30-d276-478a-8137-11558a381c05

File: public/images/pokemon/icons/variant/8/867_3.png
Comment (pom-eranian) [inline public/images/pokemon/icons/variant/8/867_3.png:1]: Doesn't match.
a2ef40bd-304c-4da1-a3a6-388bf42f8cf0

**Review by pom-eranian — COMMENTED** [review]
Commented for the few icons that 
- don't match current dex 
    - 867 runerigus  
    - 424 aipom T2

or are confirmed incorrect
- 255 torchic male T2
- 256 combusken male T2

**Comment by pom-eranian** [comment]
> Commented for the few icons that
> 
>     * don't match current dex
>       
>       * 867 runerigus
>       * 424 aipom T2
> 
> 
> or are confirmed incorrect
> 
>     * 255 torchic male T2
> 
>     * 256 combusken male T2

@chaosgrimmon confirmed torchic and combusken do line up, and is aware of the other differences. Further discussion about ambipom and runerigus here:
https://discord.com/channels/1125469663833370665/1274883292897738774/1277318467430649857

**Review by Tempo-anon — APPROVED** [review]

**Review by pom-eranian — APPROVED** [review]

**Review by flx-sta — APPROVED** [review]
Just mirroring @pom-eranian s approval

**Pull request merged**